mm2src/crypto/Cargo.toml

```diff
@@ -7,17 +7,24 @@ edition = "2018"
 doctest = false
 
 [dependencies]
+aes = "0.8.3"
```

Comment:
Q: Can we use version `0.7.5` instead of duplicating this dependency?

Reply:
I wanted to use the latest version for such an important encryption, after this https://github.com/KomodoPlatform/komodo-defi-framework/pull/1957 is merged I can try to update `0.7.5` to `0.8.3` instead in `librustzcash`